src/components/content/DocumentContent.js

```diff
@@ -0,0 +1,34 @@
+import { request } from "../../api.js";
+import Editor from "./Editor.js";
+
+export default function DocumentContent({ $target, initialId, onEditing }) {
+  const $contentSection = document.createElement("section");
+  $contentSection.setAttribute("id", "contentSection");
+
+  this.id = initialId;
+
+  this.setId = async (nextId) => {
+    this.id = nextId;
+    await fetchData();
+    this.render();
+  };
+
+  const editor = new Editor({
+    $target: $contentSection,
+    onEditing: onEditing,
```

Comment:
```suggestion
    onEditing,
```
두 인자명이 같다면 다음과같은 형태로 줄여서 사용할 수 있습니다.